fastmri_recon/data/test_generators.py

```diff
@@ -38,8 +41,53 @@ def flow_random_shapes(self,):
             mask_batch = mask_batch[..., 0]
             yield (kspaces, mask_batch), images
 
+    @threadsafe_generator
     def flow_z_filled_random_shapes(self,):
         random_shapes_gen = self.flow_random_shapes()
-        for (kspaces, _), images in random_shapes_gen:
+        for (kspaces, mask_batch), images in random_shapes_gen:
+            z_filled = zero_filled_recon(kspaces[..., 0], crop=False)[..., None]
+            yield z_filled, images, mask_batch
+
+class CifarGenerator:
+    def __init__(self, af, size, data, max_size, batch_size=1):
+        self.af = af
+        self.data = data
+        self.size = size
+        self.batch_size = batch_size
+        self.im_shape = (batch_size, size, size, 1)
+        self.index = 0
+        self.max_size = max_size
```

Comment:
isn't `max_size` a function of `data` ? same for `size`

Reply:
Indeed